src/main/java/jarvis/Jarvis.java

```diff
@@ -0,0 +1,90 @@
+package jarvis;
+
+import java.util.HashMap;
+
+import enums.Command;
+import exception.JarvisException;
+import parser.Parser;
+import storage.Storage;
+import task.TaskList;
+import ui.Ui;
+
+public class Jarvis {
+    private Ui ui;
+    private TaskList tasks;
+    private Storage storage;
+
+    private boolean processNext = true;
```

Comment:
Maybe you can consider changing processNext to a question like isProcessNext to follow coding standard?